### PR TITLE
My_calculator Milestone 2

### PR DESCRIPTION
**Pull request by Davidosky007**
## My_calculator Milestone-2
In this milestone

* components directory was created in the src folder
* four files were created inside the components directory each representing a React component
* React fragment was used to group child components when rendering
* props were passed into Display and Button components as a string

### CODE REVIEW

**Comment by Torres-ssf** [comment]
:warning: Invalid submission. 
Hello @Davidosky007 :hand: 
I need to consider this is an invalid submission since the code review request submitted was for the project Bookstore Milestone 2. We need to submit the code review request for Calculator app milestone 2

File: src/components/Display.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  <div>
+    <h2>
+      This is my calculation result:
+      {result}
+    </h2>
+  </div>
+);
+
+Display.propTypes = { result: propTypes.string.isRequired };
+
```
Comment (arslanbisharat) [inline src/components/Display.js]: - [x] its result prop should be a String with default value = 0.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested 💀 

Hello Developer,  👋 

There is always room to improve your code. Your project has met almost all the requirements but I would like to suggest some changes. I hope the next TSE who will review this project would be really impressed if you make those changes. 

**I have given an inline suggestion please have a look at it.** 

When you are done with the changes please request another review. 

Best, 
Arslan Bisharat. 
slack: arslanbisharat ☎️

**Review by mikenath223 — APPROVED** [review]
### CONGRATULATIONS  👏👏👏

This project meets the requirements for approval.

**STATUS: APPROVED ✅**


Don't forget to fill the project completion form👍
Good luck with the next project.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

**Pull request merged**